framework/src/timeintegrators/CentralDifference.C

```diff
@@ -24,17 +24,22 @@ CentralDifference::validParams()
 
   params.addClassDescription("Implementation of explicit, Central Difference integration without "
                              "invoking any of the nonlinear solver");
+  params.addParam<bool>(
+      "use_direct", false, "Whether or not to use a direct acceleration calculation");
 
   return params;
 }
 
 CentralDifference::CentralDifference(const InputParameters & parameters)
   : ActuallyExplicitEuler(parameters),
     _du_dotdot_du(_sys.duDotDotDu()),
-    _solution_older(_sys.solutionState(2))
+    _solution_older(_sys.solutionState(2)),
+    _use_direct(getParam<bool>("use_direct"))
 {
   if (_solve_type == LUMPED)
     _is_lumped = true;
+  if (_use_direct)
+    _is_direct = true;
```

Comment:
Why do you have `_is_direct` and `_use_direct`?

Reply:
`_is_direct = true` sets the variable defined in `TimeIntegrator.h` that determines whether or not the time integrator is using a direct accel. calculation. `use_direct` is a parameter in `CentralDifference.C` asking the user if they want to use a direct accel. calculation.

Reply:
So you can have `_is_direct == true` and `_use_direct == false`? That's a bit confusing. What would be the behavior in that case?

Reply:
It is a bit confusing. You couldn't have that case because `_is_direct` is not set by the user, it is just a flag to show that that the user specified `_use_direct`. 
aaand, writing that out made me realize how silly that is. I'll fix this.

Reply:
Addressed by [fd5f612](https://github.com/idaholab/moose/pull/27991/commits/fd5f61230a5171580af25121f4b5a7dc55630c62)

Reply:
Let me know if you find this fix acceptable